subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -271,7 +271,7 @@ static void isr_rx(void *param)
 	/* Check CRC and generate Periodic Advertising Report */
 	if (crc_ok) {
 		/* TODO: */
-		printk("CRC OK\n");
+		BT_WARN("CRC OK");
```

Comment:
I wonder if this is something we indeed want to have merged as is. Perhaps `BT_DBG` i more fitting, than warning us when something is OK :)

Reply:
As the feature is not complete and to avoid other debug messages, WARN is used in this PR. Shall I instead change the the message to "Periodic Event Sync-ed, report generation unsupported/work in progress." ?

Reply:
`BT_WARN` just seems to be misused here :) 

If it's for debugging purposes, it should probably be `BT_DBG`. If it's for information purposes, it should probably be `BT_INFO`.